web/hooks/use-contracts.ts

```diff
@@ -50,21 +49,6 @@ export const useInactiveContracts = () => {
   return contracts
 }
 
-export const useTaggedContracts = (tags: string[] | undefined) => {
-  const [contracts, setContracts] = useState<Contract[] | undefined>(
-    tags && tags.length === 0 ? [] : undefined
-  )
-  const tagsKey = tags?.map((tag) => tag.toLowerCase()).join(',') ?? ''
-
-  useEffect(() => {
-    if (!tags || tags.length === 0) return
-    return listenForTaggedContracts(tags, setContracts)
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [tagsKey])
-
-  return contracts
-}
-
```

Comment:
I'm not so sure about deleting this code as it seems likely to be useful in the future. Yes, it's possible we could dig through git history to revive it, but probably someone other than me will end up rewriting it.

Could we keep this and `listenForTaggedContracts`?

Reply:
Sure.